nvme-print.c

```diff
@@ -2934,9 +2940,38 @@ void json_nvme_zns_id_ns(struct nvme_zns_id_ns *ns,
 	json_free_object(root);
 }
 
+static void show_nvme_id_ns_zoned_zoc(__le16 ns_zoc)
+{
+	__u16 zoc = le16_to_cpu(ns_zoc);
+	__u8 rsvd = (zoc & 0xfc) >> 2;
+	__u8 ze = (zoc & 0x2) >> 1;
+	__u8 vzc = zoc & 0x1;
+	if (rsvd)
+		printf("  [7:2] : %#x\tReserved\n", rsvd);
+	printf("  [1:1] : %#x\tZone Excursions: %s\n",
+		ze, ze ? "Yes" : "No");
+	printf("  [0:0] : %#x\tVariable Zone Capacity: \n",
+		vzc, vzc ? "Not " : "");
```

Comment:
Make it Yes (Host support required)/No for variable zone capacity.

Reply:
oh... mistake. I didn't save the files.. haha thanks for checking it again